evm/utils/state_access_restriction.py

```diff
@@ -60,8 +67,28 @@ def to_prefix_list_form(access_list):
     """
     for obj in access_list:
         address, *storage_prefixes = obj
-        yield keccak(address) + NONCE_TRIE_PREFIX
-        yield keccak(address) + BALANCE_TRIE_PREFIX
-        yield keccak(address) + CODE_TRIE_PREFIX
+        yield nonce_key(address)
+        yield balance_key(address)
+        yield code_key(address)
         for prefix in remove_redundant_prefixes(storage_prefixes):
             yield keccak(address) + STORAGE_TRIE_PREFIX + prefix
+
+
+def storage_key(address, slot):
```

Comment:
*naming nitpick*

I had to scroll down to this module to look at  the implementation of these to find out if they were values or functions.  Could we prefix these names with something to indicate they are actions:

- `mk_storage_key`
- `construct_storage_key`
- `get_storage_key`